build.sbt

```diff
@@ -4,15 +4,19 @@ parallelExecution in ThisBuild := false
 
 val kafkaVersion = "1.1.0"
 val zookeeperVersion = "3.4.11"
+val confluentVersion = "4.1.0"
```

Comment:
Could this dependency superseed Kafka and ZooKeeper above even if ZK version is 3.4.10?

Reply:
if it doesn't clash with zookeeper used by Kafka yes